lib-cloudsdk/src/main/java/com/facebook/openwifi/cloudsdk/models/ap/State.java

```diff
@@ -132,4 +302,24 @@ public static class Radio {
 	@SerializedName("link-state") public JsonObject linkState;
 	public JsonObject gps;
 	public JsonObject poe;
+
+	/** Default Constructor with no args */
+	public State() {}
+
+	/** Copy Constructor */
+	public State(State state) {
```

Comment:
I don't understand why you had to make all these changes here -- this makes editing and reading this file very difficult. Isn't this not even being used? If you absolutely need it, why not just deep copy via `gson` (does it need to be performant?)

Reply:
Will do.

Reply:
I am still confused, where is this used? There is already a `Utils.deepCopy()` method too.

Reply:
Ah, I'm thinking of copying information from `State` to `StateInfo` when serializing a new `StateInfo` instance in `Moduler` .